src/components/single-pokemon-component.js

```diff
@@ -0,0 +1,12 @@
+export const SinglePokemonComponent = (props) => {
+  return (
+    <div>
+      <p>{props.singlePokemonData[0].name}</p>
+      <p>{props.singlePokemonData[0].id}</p>
+      <img
+        src={props.singlePokemonData[0].image}
+        alt={props.singlePokemonData[0].name}
+      />
+    </div>
+  );
+};
```

Comment:
re: https://github.com/Parkcrest/Pokedex/pull/8/files#r1068293096

```suggestion
export const SinglePokemonComponent = ({name, id, image}) => {
  return (
    <div>
      <p>{name}</p>
      <p>{id}</p>
      <img
        src={image}
        alt={name}
      />
    </div>
  );
};
```